.github/ISSUE_TEMPLATE/dswg-issue-submission-form-template.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Should this file be here?  I am having trouble spotting the differences between this one and the one below it.

Reply:
you are correct - I deleted this file, thanks!